server/routers/auth-router.js

```diff
@@ -0,0 +1,9 @@
+const { AuthHandler, UserHandler } = require('../handlers')
+const express = require('express')
+
+const router = express.Router()
+
+router.route('/login')
+  .head(UserHandler.retrievePassword, AuthHandler.login)
```

Comment:
Yes. There are many ways to do logout with web tokens. Each approach has its pros and cons. We need to talk about it.